src/services/trips/MyTripsService.ts

```diff
@@ -0,0 +1,17 @@
+/*import axios from 'axios'*/
+// eslint-disable-next-line jest/no-mocks-import
```

Comment:
Can we remove these comented lines

Reply:
I have used mocked data currently instead of backend API call. It results eslint error. Currently added that commented out line only to disable that error .
After the backed API call integration i can remove that commented out lines.